forge-gui/res/cardsfolder/upcoming/brokers_initiate.txt

```diff
@@ -0,0 +1,6 @@
+Name:Brokers Initiate
+ManaCost:W
+Types:Creature Cat Citizen
+PT:0/4
+A:AB$ Animate | Cost$ 4 GU | Defined$ Self | Power$ 5 | Toughness$ 5 | SpellDescription$ CARDNAME has base power and toughness 5/5 until end of turn.
+Oracle:{4}{G/U}: Brokers Initiate has base power and toughness 5/5 until end of turn.
```

Comment:
`DeckHints:Color$Green|Blue`

I think I need to add similar to the Maestros Initiate